DataFormats/HcalDigi/src/classes_def.xml

```diff
@@ -42,11 +42,6 @@
    <class name="HcalTTPDigi" ClassVersion="10">
     <version ClassVersion="10" checksum="3127055275"/>
    </class>
-   <class name="HcalUpgradeDataFrame" ClassVersion="12">
-    <version ClassVersion="10" checksum="2143388425"/>
```

Comment:
this is how removal was handled in another case
#15736
there the samples had more general use (affecting run2 MC)